src/Slugger.ts

```diff
@@ -1,31 +1,30 @@
+import type { SluggerOptions } from './MarkedOptions.ts';
+
 /**
  * Slugger generates header id
  */
-export class Slugger {
+export class _Slugger {
+  seen: { [slugValue: string]: number };
+
   constructor() {
     this.seen = {};
   }
 
-  /**
-   * @param {string} value
-   */
-  serialize(value) {
+  serialize(value: string) {
     return value
       .toLowerCase()
       .trim()
-      // remove html tags
+    // remove html tags
       .replace(/<[!\/a-z].*?>/ig, '')
-      // remove unwanted chars
+    // remove unwanted chars
```

Comment:
Looks like indentation changed unexpectedly

Reply:
🤷‍♂️ Must have been the IntelliJ internal formatter. Tried to revert those